example/standalone_git/README.md

```diff
@@ -3,20 +3,69 @@
 This example deploys a NiFi container connected to a NiFi Registry container with git integration (Github).
 Before running the example, these variables must be set beforehand.
 
-| Variable   | Default value | Description |
-| ------------- | ------------- | ------------- |
-| `git_remote_url`  | " "  | URL of the remote git repository |
-| `git_checkout_branch`  | " "  |Branch to checkout and track.|
-| `git_flow_storage_directory`  | /opt/nifi-registry/flow-storage  | Default value is set by image, original default value was "./flow-storage". This variable is used in clone script as a destination directory for clone. It is also used in NiFi Registry configuration.|
-| `git_remote_to_push`  | origin  |This variable is used in clone script to set origin name using `-o, --origin <name> use <name> instead of 'origin' to track upstream`. It is also used in NiFi Registry configuration.|
-| `git_access_user`  | " "  |Username|
-| `git_access_password`  | " "  |Password|
-| `git_user_name`  | nifi-registry  |Developer|
-| `git_user_email`  | nifi-registry@localhost  |Developer email|
-
-__NOTE!__ In this version of the example, the variables above __must be written directly in the [main.tf](main.tf#L52-L59)__.
-We are still improving this workflow to be more secure and agile.
+| Variable   |  Default value |Defined from CLI | Description |
+| ------------- | ------------- | ------------- | ------------- |
+| `git_remote_url`  | " "  | Yes |URL of the remote git repository. Use HTTPS. |
+| `git_checkout_branch`  | " "  | Yes| Branch to checkout and track.|
+| `git_flow_storage_directory`  | /opt/nifi-registry/flow-storage  | No| Default value is set by image, original default value was "./flow-storage". This variable is used in clone script as a destination directory for clone. It is also used in NiFi Registry configuration.|
+| `git_remote_to_push`  | origin  | No |This variable is used in clone script to set origin name using `-o, --origin <name> use <name> instead of 'origin' to track upstream`. It is also used in NiFi Registry configuration.|
+| `git_access_user`  | " "  | Yes | GitHub Username |
+| `git_access_password`  | " "  | Yes | [Personal token from GitHub](https://docs.github.com/en/github/authenticating-to-github/creating-a-personal-access-token)|
```

Comment:
In `terraform-nomad-nifiregistry` we added a [sentence that we recommend our users](https://github.com/hannemariavister/terraform-nomad-nifiregistry/blob/master/example/standalone_git/README.md) to create a new token for this purpose. Maybe we should do the same her?

Reply:
Done